content/en/docs/concepts/configuration/configmap.md

```diff
@@ -212,9 +212,9 @@ when new keys are projected to the Pod can be as long as the kubelet sync period
 propagation delay, where the cache propagation delay depends on the chosen cache type
 (it equals to watch propagation delay, ttl of cache, or zero correspondingly).
 
-{{< feature-state for_k8s_version="v1.18" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.18" state="beta" >}}
```

Comment:
Hi @wojtek-t .
For ConfigMap, should the k8s version move to `v1.19`?

Reply:
good catch -done